h/static/scripts/bridge.coffee

```diff
@@ -14,20 +14,28 @@ module.exports = class Bridge
     @channelListeners = {}
     @onConnectListeners = []
 
-  createChannel: (source, origin, scope) ->
+  createChannel: (source, origin, token) ->
+    channel = null
+    connected = false
+
+    ready = =>
+      return if connected
+      connected = true
+      for cb in @onConnectListeners
+        cb.call(null, channel, source)
```

Comment:
I'm guessing nobody uses the function binding value (which has changed from the bridge instance to null).